architecture/routing.adoc

```diff
@@ -85,18 +119,34 @@ Edge termination is configured by setting `tls:termination` to `edge` on your ro
     "host": "www.example.com",
     "serviceName": "hello-nginx",
     "tls": {
-        "termination": "edge",
-        "certificate": "<Certificate_File>",
-        "key": "<Key_File>",
-        "caCertificate": "<CaCertificate_File>"
+        "termination": "edge", <1>
+        "certificate": "<cert_text>", <2>
+        "key": "<cert_text>", <3>
+        "caCertificate": "<cert_text>" <4>
      }
 }
 ----
```

Comment:
I have thoughts on this, but I may not be able to put them into words correctly. I saw there was a conversation about this on IRC when I was away, but I'll express thoughts anyway. 

I can see what this is achieving, and I agree with it, but I'm not sure if this is up to the guidelines that have been set. Firstly, I don't think this can be an example if we're not putting the actual values in. (But I'm not sure if it fits into code block either.) However, I get that they're very long values, and that's why these have been put into replaceables.

Secondly, I know the replaceable values are still being ironed out. But the only one of these we have left from v2 is Cert_File. Would it be too picky if this was changed to have caps, eg: Cert_file? I'm also not sure of the difference between Cert_File and Cert_Text.

Other than that, I agree with what's been done.